menu.ipxe

```diff
@@ -64,6 +65,12 @@ initrd ${base-url}/coreos_production_pxe_image.cpio.gz
 boot
 goto MENU
 
+:core-6.4
+kernel core-6.4/vmlinuz --- console=ttyS0,115200 earlyprint=serial,ttyS0,115200 
```

Comment:
@pietrushnic I assume the relevant kernel and initrd are uploaded?

Reply:
@miczyg1 yes, here: https://cloud.3mdeb.com/index.php/s/AQuUdsYkBzO9UJz I also updated ansible to add this package automatically to pxe-server.